lib/associations/belongs-to-many.js

```diff
@@ -287,6 +287,16 @@ class BelongsToMany extends Association {
        */
       removeMultiple: 'remove' + plural,
       /**
+       * Un-associate all instances.
+       *
+       * @param {Object} [options] Options passed to `through.destroy`
+       * @return {Promise}
+       * @method removeAssociations
+       * @memberof Associations.BelongsToMany
+       * @instance
+       */
+      destroy: 'destroy' + plural,
```

Comment:
Since there is `removeAssociation(instance)`, I would expect a `destroyAssociation(instance)` to exist too